docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+
+### 1.4.6. Welke bakjes onderkennen we?
+
+In de uitwerking hierboven hebben we al enkele registraties (‘bakjes’) naast de ZRC onderscheiden. Een aparte registratie met besluiten, een aparte registratie voor documenten (bestaat nu vaak al), een aparte registratie voor klanten en klantcontacten,…
+Dit is echter slechts een eerste aanzet en nog niet volledig uitgewerkt. Willen we voor iedere “object” een aparte registratie?
+Opmerking AK  Benoem domeinen, werk de informatievoorzienin daarvan uit in informatiemodellen en je hebt de objecttypen, registraties en bakjes. Zo simpel is dat …
+Of horen sommige objecten echt bij elkaar? Wat is de relatie met het ImZTC? Om de eenduidigheid van de implementaties ten goede te komen, is het nodig om dit verder uit te werken.  Hoe vertalen we de objecten uit het RGBZ naar registraties in een gegevenslandschap en hoe relateren we deze aan elkaar? Met het informatiemodel laten we de implementatie nadrukkelijk vrij.
+In een gegevenslandschap is het wenselijk om dit verder uit te werken in een gegevensmodel. En wellicht ook met een concreet voorbeeld om dit concept aan iedereen goed duidelijk te maken.
+
+### 1.4.7. Wat willen we doen met Zaak?
+
+Zaakgericht werken kent al een lange geschiedenis. Het stamt uit de tijd dat dienstverleningsambities hoger werden en backoffice-applicaties dit niet of nauwelijks aankonden. Daarna was er de gedachte dat een “alles-in-1 zaaksysteem” voor veel processen voldoende zou zijn. Inmiddels draait de slinger een beetje terug en zien we taakspecifieke systemen moderner en opener worden. In een gegevenslandschap is dit helemaal het geval.
+
+In de loop der jaren zijn er diverse doelen voor zaakgericht werken ontstaan: dienstverlening verbeteren, als basis voor het archiveren, het bieden van managementinformatie,.. waarvan het de vraag is of je ze in een modern applicatielandschap nog met zaakgericht werken wilt oplossen. In het geval van een gemeente die met één dominant generiek zaaksysteem werkt ligt de oplossingsrichting voor de hand. In een gegevenslandshap veel minder. Bovendien zien we op vele vlakken een terugtrekkende beweging op het “gemeentebrede” dienstverlenigsconcept, meer richting domeinspecifieke Apps en portalen. Een meer domeingerichte dienstverlening en informatievoorziening (ipv generiek obv zaken) ligt dan meer voor de hand. Opmerking AK  Die twee zijn niet strijdig. Het lijkt er op dat in de redenering  en de voorbeelden het zaakgericht werken als panacee voor alle problemen en behoeften wordt gezien. Evenwel, zaakgericht werken is niet mee of minder dan een manier van procesgericht werken. Een manier die zo ver is uitgewerkt dat anderen dan de uitvoerenden van een proces dat proces ook kunnen begrijpen, op het niveau dat voor een ander volstaat: de hoofdlijnen. Dit ‘esperanto’ blijft een onderscheidend kenmerk van zaakgericht werken en zaken. Je krijgt daarmee twee niveau’s: domeinspecifiek en domeinoverstijgend. Het tweede doen we met zaken.
```

Comment:
Ik weet niet tot hoever de opmerking van AK doorloopt maar ik vind vanaf "Evenwel, zaakgericht werken is..." t/m het einde van de paragraaf, een erg helder stukje!